styles.css

```diff
@@ -1,7 +1,73 @@
-h1 {
-  color: red;
+.flex-container {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-p {
-  color: blue;
+nav {
+  height: 80px;
+  width: 100%;
+  font-size: 30px;
+}
+
+nav label {
+  float: right;
+  line-height: 80px;
+  position: relative;
+  top: 50px;
+  right: 70px;
+}
+
+.logo {
+  position: relative;
+  float: left;
+  top: 70px;
+  left: 20px;
+  font-weight: bold;
+}
+
+section h1 {
+  color: #172b4d;
+  text-align: center;
+  margin: 30px;
+  margin-top: 140px;
+  font-size: 40px;
+  font-weight: bold;
+}
+
+section h2 {
+  text-align: center;
+  margin: 30px;
+  font-size: 40px;
+  font-weight: bold;
+  color: #36b37e;
+}
+
+section p {
+  color: #172b4d;
+  text-align: center;
+  text-indent: 28px;
+  font-size: 24px;
+  font-weight: bold;
+  font-family: 'inter';
+  font-style: normal;
+  line-height: 28px;
+  margin: 30px;
+  background-color: transparent;
+}
```

Comment:
- [ ] Great job so far. Your text is aligned in the center and that's great 👍. However, it looks like you have not styled it very well. Try to look at this comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/176922035-34f45f54-f143-454c-b663-6310c68fe91a.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/176922750-299b68df-221b-46ff-bd81-824eeae70617.png)
 You can see that the `font-weight` is different. As a developer, it is important to pay attention to all the details even the small ones because it is what makes you outstanding 😃. Please visit the Figma design and check the right styles used.
💡 **HINTS**:
- It might be Inter, sans-serif
- Font-weight is not bold, so consider removing it